examples/profile/controls/example.rb

```diff
@@ -1,23 +1,35 @@
-# copyright: 2015, Chef Software, Inc.
+# copyright: 2016, Chef Software, Inc.
 
-title '/tmp profile'
+title 'Example Config Checks'
 
-# you add controls here
-control "tmp-1.0" do                                   # A unique ID for this control
-  impact 0.7                                           # The criticality, if this control fails.
-  title "Create /tmp directory"                        # A human-readable title
-  desc "An optional description..."                    # Describe why this is needed
-  desc "label", "An optional description with a label" # Pair a part of the description with a label
-  tag data: "temp data"                                # A tag allows you to associate key information
-  tag "security"                                       # to the test
-  ref "Document A-12", url: 'http://...'               # Additional references
+# To pass the test, create the following file
+# ```bash
+# mkdir -p /tmp/example
+# cat <<EOF > /tmp/example/config.yaml
+# version: '1.0'
+# EOF
+# ```
+control 'example-1.0' do
+  impact 'critical'
+  title 'Verify the version number of Example'
+  desc 'An optional description...'
+  tag 'example'
+  ref 'Example Requirements 1.0', uri: 'http://...'
```

Comment:
It's easier to track this stuff if you have a commit for JUST renames and then do content changes after.